spec/integration/estimates_spec.rb

```diff
@@ -99,4 +99,37 @@
     expect(bitcoin_estimate.data.type).to eq 'bitcoin'
     expect(bitcoin_estimate.data.mass_g).to be < bitcoin_estimate_2.data.mass_g
   end
+
+  it 'supports creating bitcoin estimates with a timestamp' do
```

Comment:
the sdk code is autogenerated but I'm assuming the tests are written manually. Can you please confirm?

Reply:
it would be nice to wrap each group of `it` for a method being tested in a `describe` block. On a second take, I realized this test was for the bitcoin method and not the ethereum method

Reply:
Yes. Code is auto-generated, tests are manual.

Good call on the grouping, I'll update this PR.